doom-modeline-segments.el

```diff
@@ -771,12 +771,12 @@ level."
                 ('finished  (if flycheck-current-errors
                                 (let-alist (doom-modeline--flycheck-count-errors)
                                   (doom-modeline-checker-icon
-                                   "block" "🚫" "!"
+                                   "error_outline" "🚫" "!"
```

Comment:
Should unicode char fallback be changed to "❗" accordingly?

Reply:
I'm not sure. I was thinking of doing so, but I noticed that was already used for another state.